src/views/Order.vue

```diff
@@ -0,0 +1,60 @@
+<template>
```

Comment:
This view is the order view which grabs all of the order's information from the store